### PR TITLE
do not use tuple for one needed arg

### PR DESCRIPTION
**Pull request by hkiel**

### CODE REVIEW

File: Compiler/BackEnd/BackendDAEOptimize.mo [hunk 1]
```diff
@@ -1152,21 +1152,21 @@ end removeUnusedParameter;
...
 protected function copyNonParamVariables
   input BackendDAE.Var inVar;
-  input tuple<BackendDAE.Variables,BackendDAE.Variables> inTpl;
+  input BackendDAE.Variables inVars;
   output BackendDAE.Var outVar;
-  output tuple<BackendDAE.Variables,BackendDAE.Variables> outTpl;
+  output BackendDAE.Variables outVars;
 algorithm
-  (outVar,outTpl) := matchcontinue (inVar,inTpl)
+  (outVar,outVars) := matchcontinue (inVar,inVars)
```
Comment (lochel) [inline Compiler/BackEnd/BackendDAEOptimize.mo]: This should be a match statement: `(outVar,outVars) := match inVar`.

File: Compiler/BackEnd/BackendDAEOptimize.mo [hunk 2]
```diff
@@ -1152,21 +1152,21 @@ end removeUnusedParameter;
...
     local
       BackendDAE.Var v;
       BackendDAE.Variables vars,vars1;
       DAE.ComponentRef cr;
-    case (v as BackendDAE.VAR(varKind = BackendDAE.PARAM()),(_,_))
-      then (v,inTpl);
-    case (v as BackendDAE.VAR(),(vars,vars1))
+    case (v as BackendDAE.VAR(varKind = BackendDAE.PARAM()),_)
+      then (v,inVars);
+    case (v as BackendDAE.VAR(),_)
```
Comment (lochel) [inline Compiler/BackEnd/BackendDAEOptimize.mo]: This case could be changed to an else branch.

File: Compiler/BackEnd/BackendDAEOptimize.mo (hunk 1 above)
Comment (hkiel) [inline Compiler/BackEnd/BackendDAEOptimize.mo]: fixed

File: Compiler/BackEnd/BackendDAEOptimize.mo (hunk 2 above)
Comment (hkiel) [inline Compiler/BackEnd/BackendDAEOptimize.mo]: fixed

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2017-04-07_17-09-15](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4169/console).

**Pull request closed without merging**